### PR TITLE
Java 1.5 compatibility fixes.

### PR DESCRIPTION
**Pull request by dbwiddis**

### CODE REVIEW

**Comment by dbwiddis** [comment]
Ugh.... test did fine but it's failing on linux due to virtual filesystem showing 0 length....

**Comment by dbwiddis** [comment]
there...

File: src/main/java/oshi/util/FileUtil.java [hunk 1]
```diff
@@ -21,10 +22,24 @@
...
+		BufferedReader br = null;
+		try {
+			br = new BufferedReader(new FileReader(filename));
+			for (;;) {
+				String line = br.readLine();
+				if (line == null)
+					break;
+				result.add(line);
+			}
+		} catch (FileNotFoundException e) {
```
Comment (dblock) [inline src/main/java/oshi/util/FileUtil.java]: Minor: hiding a `FileNotFoundException` seems unnecessary here.

**Comment by trejkaz** [comment]
New code uses the default platform charset, whereas the old code hard-coded UTF-8. Intentional?

File: src/main/java/oshi/util/FileUtil.java (hunk 1 above)
Comment (dbwiddis) [inline src/main/java/oshi/util/FileUtil.java]: That was residual from implementing one step at a time, and catching that before later deciding to just throw IOExceptions.

**Comment by dbwiddis** [comment]
@trejkaz Good question.  Not intentional, but after researching I can't find where /proc filesystem differs in encoding from the system default. For our purposes (reading /proc and /sys) the information is all ASCII which is common across charsets.

**Pull request merged**